scout/parse/variant/transcript.py

```diff
@@ -48,26 +39,10 @@ def parse_transcripts(raw_transcripts, allele=None):
         transcript["protein_id"] = entry.get("ENSP")
 
         ## Polyphen prediction ##
-        polyphen_prediction = entry.get("POLYPHEN")
-        # Default is 'unknown'
-        prediction_term = "unknown"
-        if polyphen_prediction:
-            prediction_term = polyphen_prediction.split("(")[0]
-        transcript["polyphen_prediction"] = prediction_term
+        transcript["polyphen_prediction"] = get_prediction_term(entry)
```

Comment:
Just for uniformity with the SIFT below.
```suggestion
        transcript["polyphen_prediction"] = get_polyphen_prediction(entry)
```